Error running `kimai:bundle:demo:install` on new installation

I set kimai up by following [the documentation (development install)](https://www.kimai.org/documentation/developers.html#development-installation), and kimai (in dev environment) is working fine.

Then I cloned this repo into var/plugins. When I run `bin/console kimai:bundle:demo:install`, I get following error:
```
❯ bin/console kimai:bundle:demo:install

Starting installation of plugin: DemoBundle ...
===============================================

                                                                                                                        
 [ERROR] Failed to install database for bundle DemoBundle. Migrations configuration key "name" does not exist.
```

Commenting out the line just moves the error to `"table_name" does not exist`

--- a/Migrations/demo.yaml
+++ b/Migrations/demo.yaml
@@ -7,7 +7,8 @@
 # bin/console doctrine:migrations:migrate --em=default --configuration=var/plugins/DemoBundle/Migrations/demo.yaml
 # --------------------------------------------------------------------------------------------------
 
-name: DemoMigrations
-table_name: bundle_migration_demo
-migrations_namespace: DemoBundle\Migrations
-migrations_directory: var/plugins/DemoBundle/Migrations
+table_storage:
+    table_name: bundle_migration_demo
+
+migrations_paths:
+    DemoBundle\Migrations: var/plugins/DemoBundle/Migrations

